test/scenario4.js

```diff
@@ -0,0 +1,209 @@
+    guardianTwoSc = new SafientSDK(guardianTwoSigner, chainId);
+    guardianTwo = await guardianTwoSc.safientCore.connectUser(apiKey, secret);
+    // SUCCESS : create user A
+    const userAddress = await guardianTwoSigner.getAddress()
+    await guardianTwoSc.safientCore.registerNewUser('Guardian 2', 'guardianTwo@test.com', 0, userAddress);
+
+    // FAILURE : try creating user A again
+    await expect(guardianTwoSc.safientCore.registerNewUser('Guardian 2', 'guardianTwo@test.com', 0, userAddress)).to.eventually.be.eql('guardianTwo@test.com already registered.');
+
+    // SUCCESS : get all users (check if the user A was created)
+    const loginUser = await guardianTwoSc.safientCore.getLoginUser(guardianTwo.idx.id);
+    expect(loginUser.name).to.equal('Guardian 2');
+    expect(loginUser.email).to.equal('guardianTwo@test.com');
+});
+
+it('Should register a Guardian 3', async () => {
+    guardianThreeSc = new SafientSDK(guardianThreeSigner, chainId);
+    guardianThree = await guardianThreeSc.safientCore.connectUser(apiKey, secret);
+
+    const userAddress = await guardianThreeSigner.getAddress()
+    await guardianThreeSc.safientCore.registerNewUser('Guardian 3', 'guardianThree@test.com', 0, userAddress);
+
+    // FAILURE : try creating user A again
+    await expect(guardianThreeSc.safientCore.registerNewUser('Guardian 3', 'guardianThree@test.com', 0, userAddress)).to.eventually.be.eql('guardianThree@test.com already registered.');
+
+    // SUCCESS : get all users (check if the user A was created)
+    const loginUser = await guardianThreeSc.safientCore.getLoginUser(guardianThree.idx.id);
+    expect(loginUser.name).to.equal('Guardian 3');
+    expect(loginUser.email).to.equal('guardianThree@test.com');
+});
+
+  //should create a safe onChain and offChain
+  it('Should create safe with "Testing Safe data" with Signal Based Claim', async () => {
+   
+     
+      safeId = await creatorSc.safientCore.createNewSafe(creator.idx.id, beneficiary.idx.id, "Testing safe Data", true, 0, 6)
+      const safeData = await creatorSc.safientCore.getSafeData(safeId);
+      expect(safeData.creator).to.equal(creator.idx.id);
+  });
+
+
+  //Step 3: Create a claim
+  it('Should create a claim', async () => {
+    const file = {
+        name: "signature.jpg"
+    }
+    disputeId = await beneficiarySc.safientCore.claimSafe(safeId, file, "Testing Evidence", "Lorsem Text")
+    expect(disputeId).to.be.a('number');
+  });
+
+  it('Should send signal after claim', async () => {
+
+      const result = await creatorSc.safientCore.sendSignal(safeId) //Passing a claim
+      expect(result.status).to.equal(1);
+  });
+
+  it('Should create a claim', async () => {
+    const file = {
+        name: "signature.jpg"
+    }
+    disputeId = await beneficiarySc.safientCore.claimSafe(safeId, file, "Testing Evidence", "Lorsem Text")
+    expect(disputeId).to.be.a('number');
+  });
+
+it('Should wait for somtime', async () => {
```

Comment:
Don't make this is a separate test. You can add the delay in the previous test itself.